File: chapter1. 오브젝트와 의존관계/chapter1_tlswltjq/src/main/java/org/example/chapter1_tlswltjq/dao/SimpleConnectionMaker.java
package org.example.chapter1_tlswltjq.dao;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

public class SimpleConnectionMaker {
    public Connection makeNew() throws ClassNotFoundException, SQLException {
        Class.forName("org.h2.Driver");
        Connection c = DriverManager.getConnection("jdbc:h2:~/toby", "sa", "");
        return c;
    }
}

File: chapter1. 오브젝트와 의존관계/chapter1_albireo3754/src/main/kotlin/SimpleConnectionMaker.kt
package com.albireo3754

import java.sql.Connection
import java.sql.DriverManager

class SimpleConnectionMaker {
    fun getConnection(): Connection =
        DriverManager.getConnection(
            "jdbc:mysql://localhost:3308/springbook?createDatabaseIfNotExist=true",
            "root",
            null
        )
}